internal/plugins/ansible/templates/config/rbac/kustomization.go

```diff
@@ -0,0 +1,113 @@
+/*
+TODO(asmacdo) Modified
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rbac
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"sigs.k8s.io/kubebuilder/pkg/model/file"
+)
+
+var _ file.Template = &KustomizeRBAC{}
+
+var rbacKustomizePath = filepath.Join("config", "rbac", "kustomization.yaml")
+
+const patch6902Marker = "patch6902"
+
+// KustomizeRBAC scaffolds the Kustomization file in rbac folder.
+type KustomizeRBAC struct {
+	file.TemplateMixin
+}
+
+// SetTemplateDefaults implements input.Template
+func (f *KustomizeRBAC) SetTemplateDefaults() error {
+	if f.Path == "" {
+		f.Path = rbacKustomizePath
+	}
+
+	f.TemplateBody = fmt.Sprintf(kustomizeTemplate,
+		file.NewMarkerFor(f.Path, patch6902Marker),
+	)
+
+	// TODO(asmacdo)
+	// f.IfExistsAction = file.Error
+
+	return nil
+}
+
+type KustomizeUpdater struct {
+	file.TemplateMixin
+	file.ResourceMixin
+}
+
+func (*KustomizeUpdater) GetIfExistsAction() file.IfExistsAction {
+	return file.Overwrite
+}
+
+func (*KustomizeUpdater) GetPath() string {
+	return rbacKustomizePath
+}
+
+func (f *KustomizeUpdater) GetMarkers() []file.Marker {
+	return []file.Marker{
+		file.NewMarkerFor(rbacKustomizePath, patch6902Marker),
+	}
+}
+
+func (f *KustomizeUpdater) GetCodeFragments() file.CodeFragmentsMap {
+	fragments := make(file.CodeFragmentsMap, 1)
+
+	// If resource is not being provided we are creating the file, not updating it
+	if f.Resource == nil {
+		return fragments
+	}
+
+	// Generate patch6902 fragments
+	patches := make([]string, 0)
+	patches = append(patches, f.Resource.Replacer().Replace(patch6902Fragment))
```

Comment:
To update the scaffolds and followup the current design we should use markers instead.